seqr/views/apis/data_manager_api_tests.py

```diff
@@ -816,6 +816,14 @@ def mock_write(content):
                 mock_open.assert_called_with(file_name, 'wt')
                 self.assertListEqual(mock_writes, [row.replace(PLACEHOLDER_GUID, new_sample_guid) for row in params['parsed_file_data']])
 
+                # test loading new data without existing data
+                data = [params['new_data'][0]]
+                data[0][0] = 'NA19678'  # load data for a new individual
+                _set_file_iter_stdout([header] + data)
+                body.pop('mappingFile')
+                response = self.client.post(url, content_type='application/json', data=json.dumps(body))
+                self.assertEqual(response.status_code, 200)
```

Comment:
You need to add tests that this is actually doing what we want, not just that it has a successful status code. Add a test for the `response_json` being what we expect and use `_check_rna_sample_model` to test the database mode is correct

Reply:
When the bug happens, there is an uncaught exception. So the `response.status_code` will be 500. For testing the bug itself, it is enough to test the status code. But I will add the model check to ensure the change is correct.

Reply:
tests are to test functionality for all changes going forward, not just one bug this one time. What if someone changed fixture data and this no longer was doing what we wanted it to do and was instead returning a 200 response with a warning saying everything in the file was skipped? How would we know from this test that we had lost coverage?